tests/v2/test_waku_keepalive.nim

```diff
@@ -48,4 +48,6 @@ procSuite "Waku Keepalive":
     check:
       (await completionFut.withTimeout(5.seconds)) == true
 
-    await allFutures([node1.stop(), node2.stop()])
+    # await allFutures([node1.stop(), node2.stop()])
```

Comment:
Is there any reason for keeping this code commented? Just asking.

Reply:
Good question. No, I commented it out as an experiment and forgot to remove when the experiment worked. :) Will address subsequently, as I don't want the CI chain to kick off again before I can merge. :)

Reply:
And you're right - we shouldn't have dead code and legacy comments cluttering the code.

Reply:
It happens to me often, too 😆